### ISSUE TITLE
Linux version → Menu useless items

### ISSUE THREAD

**Issue by timur-davletshin** [wireapp/wire-desktop#110]
Current Linux version (Electron based) has got a lot of useless menu items.
Steps to reproduce:
1. Archive all conversations
2. Any Conversation → (Ping, Call, Video Call, People, Add People..., Archive, Mute, Delete) does nothing.
3. Edit → (All items) useless but still active (clickable).
4. Window → (All items) useless. Name "Windows" is not representative because where are no actual windows.

IMO: This menu bar can be deleted. All functionality is available anyway.

**Comment by lipis** [comment]
Thank you for all the details! We'll have to spend some time on this one and most likely we're going to refactor the system menu..

**Comment by ConorIA** [comment]
I've just tried these menu items and they all work. By useless do you mean that they are redundant, or they really aren't working on your machine?

Or do you mean once you've Archived all conversations? Because I think it makes sense that conversation-specific commands wouldn't work once they've all been archived. 

I do agree that the "Windows" menu is a misnomer. My suggetion would be to change it to 'View'. The 'previous' and 'next' conversation itesm can probably be moved to the 'conversation' menu.

**Comment by sonicou** [comment]
On Debian using the provided .deb package, the menus don't work either. 
I take it @ConorIA uses Arch, so there seems to be a difference in the packaging, maybe a missed dependency in the Debian package?

**Comment by ConorIA** [comment]
I've recently started using electron-builder to package on Arch, albeit as an unpacked directory which I then package separately. What version are you running? 

I just tried again, all functions in the conversation menu are working. You're right though that the edit menu isn't useful. I think it is because you lose focus from the text entry box when you click the menu. This is my guess because the shortcuts Ctrl-z etc. still work when you are typing.
